projects/canopy/src/lib/promo-card/promo-card.notes.ts

```diff
@@ -1,6 +1,4 @@
 export const notes = `
-# Promo Card List Component
-
 ## Purpose
```

Comment:
We've also been removing the **Purpose** header on other stories, as it's still just as readable without.

Reply:
done 👍